api/comments.rb

```diff
@@ -1,5 +1,8 @@
 get "#{APIPREFIX}/comments/:comment_id" do |comment_id|
-  comment.to_hash(recursive: bool_recursive).to_json
+  @comment = comment
+  comment_hash = @comment.to_hash(recursive: bool_recursive)
+  verify_or_fix_cached_comment_count(@comment, comment_hash)
```

Comment:
@wajeeha-khalid It sounds like this may not be the right place for this.  Is there some call (or some parameter on this call) for which we do retrieve the children?  What is the call we use when we do want to load the comments for a response?  It is there that I want us to verify_or_fix_cached_comment_count, because we will already have read the children and can compare against the cached count.  Let me know if this makes sense.